tests/tx/test_verification.py

```diff
@@ -342,8 +342,6 @@ def test_merge_mined_block_verify_without_storage(self) -> None:
         verify_data_wrapped = Mock(wraps=self.verifiers.block.verify_data)
         verify_sigops_output_wrapped = Mock(wraps=self.verifiers.vertex.verify_sigops_output)
 
-        verify_aux_pow_wrapped = Mock(wraps=self.verifiers.merge_mined_block.verify_aux_pow)
```

Comment:
Don't you have to add this mock and `assert_called_once()` to the moved where the call was moved to?

Reply:
It's already there, since `verify` and `validate_full` called `verify_without_storage`, this was already being tested: https://github.com/HathorNetwork/hathor-core/blob/da32f8092a1ae038046864a4edc6e448b9e1b811/tests/tx/test_verification.py#L409

Reply:
It is actually the `verify_aux_pow` (not the `verify_pow`). But I checked and it's there. Thanks!